music21/converter/subConverters.py

```diff
@@ -806,19 +804,22 @@ def runThroughMusescore(self, fp, subformats=None, **keywords):
         fpOut = fp[0:len(fp) - 3]
         fpOut += subformatExtension
 
-        musescoreRun = '"' + musescorePath + '" ' + fp + " -o " + fpOut + " -T 0 "
+        musescoreRun = '' + musescorePath + ' ' + fp + " -o " + fpOut + " -T 0 "
```

Comment:
this seems quite dangerous together with the split below.  Many users have spaces in their musescorePath and their fp, fpOut paths -- if using subprocess, instead of concationating and splitting, we should build the array directly.  Make sure that this commit works for paths involving spaces.